Samples/iOS-Swift/iOS-SwiftUITests/UIEventTests.swift

```diff
@@ -0,0 +1,41 @@
+import XCTest
+
+class UIEventTests: XCTestCase {
+    private let app: XCUIApplication = XCUIApplication()
+
+    override func setUp() {
+        super.setUp()
+        continueAfterFailure = false
+        XCUIDevice.shared.orientation = .portrait
+        app.launchEnvironment["io.sentry.ui-test.test-name"] = name
+        app.launch()
+
+        waitForExistenceOfMainScreen()
+    }
+
+    override func tearDown() {
+        app.terminate()
+        super.tearDown()
+    }
+
+    func testUIEventIgnoreTextFieldChange() {
```

Comment:
`l`: I think this test name could reflect better what it is testing. Something like this would be more descriptive:
```suggestion
    func testNoBreadcrumbForTextFieldEditingChanged() {
```